inc/type.class.php

```diff
@@ -1928,6 +1928,13 @@ static function install(Migration $migration) {
 
       $notepad = new Notepad();
       foreach ($allGenericObjectTypes as $genericObjectType => $genericObjectData) {
+         $itemtype = $genericObjectData['itemtype'];
+         if (! class_exists($itemtype, true)) {
+            $warning = "Unable to load the class '$itemtype'\n";
+            $warning .="Please check the directory " . GENERICOBJECT_DOC_DIR . ""; 
```

Comment:
Should not be translatable?
Also, I wonder is this is clear enough; because user won't know what to check.

I'd be OK to add a link to the doc, but we'll have ot add a parameter for the docs main URI (and we'll have to define wich doc and wich language); and the required checks have to be added in a new doc section as anything exists right now.